src/CompanyService.Data.Provider.MsSql.Ef/Migrations/2022032820150000_Initial.cs

```diff
@@ -0,0 +1,124 @@
+          Name = table.Column<string>(nullable: false),
+          Description = table.Column<string>(nullable: true),
+          Tagline = table.Column<string>(nullable: true),
+          Contacts = table.Column<string>(nullable: true),
+          LogoContent = table.Column<string>(nullable: true),
+          LogoExtension = table.Column<string>(nullable: true),
+          CreatedBy = table.Column<Guid>(nullable: false),
+          CreatedAtUtc = table.Column<DateTime>(nullable: false),
+          ModifiedBy = table.Column<Guid>(nullable: true),
+          ModifiedAtUtc = table.Column<DateTime>(nullable: true)
+        },
+        constraints: table =>
+        {
+          table.PrimaryKey("PR_Companies", x => x.Id);
+        });
+    }
+
+    private void CreateCompaniesUsersTable(MigrationBuilder migrationBuilder)
+    {
+      migrationBuilder.CreateTable(
+        name: DbCompanyUser.TableName,
+        columns: table => new
+        {
+          Id = table.Column<Guid>(nullable: false),
+          CompanyId = table.Column<Guid>(nullable: false),
+          UserId = table.Column<Guid>(nullable: false),
+          ContractSubjectId = table.Column<Guid>(nullable: true),
+          ContractTermType = table.Column<int>(nullable: false),
+          Rate = table.Column<decimal>(nullable: true),
+          StartWorkingAt = table.Column<DateTime>(nullable: false),
+          EndWorkingAt = table.Column<DateTime>(nullable: true),
+          Probation = table.Column<DateTime>(nullable: true),
+          CreatedBy = table.Column<Guid>(nullable: false),
+          CreatedAtUtc = table.Column<DateTime>(nullable: false),
+          ModifiedBy = table.Column<Guid>(nullable: true),
+          ModifiedAtUtc = table.Column<DateTime>(nullable: true),
+          IsActive = table.Column<bool>(nullable: false)
+        },
+        constraints: table =>
+        {
+          table.PrimaryKey("PR_CompaniesUsers", x => x.Id);
+        });
+    }
+
+    private void CreateCompaniesChangesTable(MigrationBuilder migrationBuilder)
+    {
+      migrationBuilder.CreateTable(
+        name: DbCompanyChanges.TableName,
+        columns: table => new
+        {
+          Id = table.Column<Guid>(nullable: false),
+          CompanyId = table.Column<Guid>(nullable: false),
+          ModifiedBy = table.Column<Guid>(nullable: true),
+          ModifiedAtUtc = table.Column<DateTime>(nullable: true),
+          Changes = table.Column<string>(nullable: false)
+        },
+        constraints: table =>
+        {
+          table.PrimaryKey("PR_CompaniesChanges", x => x.Id);
+        });
+    }
+
+    private void CreateContractSubjectsTable(MigrationBuilder migrationBuilder)
+    {
+      migrationBuilder.CreateTable(
+        name: DbContractSubject.TableName,
+        columns: table => new
+        {
+          Id = table.Column<Guid>(nullable: false),
+          CompanyId = table.Column<Guid>(nullable: false),
+          Name = table.Column<string>(nullable: false),
```

Comment:
unique constraint

Reply:
on name? do we want to add it to db? we can check it is unique in validator